scripts/generator/graphql_generator/csharp/Editor/PostProcessor.cs.erb

```diff
@@ -0,0 +1,15 @@
+namespace <%= namespace %>.SDK {
+    using UnityEngine;
+    using UnityEditor;
+    using UnityEditor.Callbacks;
+
+    public class PostProcesser {
+
+        [PostProcessBuildAttribute(0)]
```

Comment:
Since the our process happens first, the settings can be overwritten by their custom processors. The only case where it would conflict is their process changes the iOS Target Version  below 9.0 after our runs. For developers who have experience with native plugins, they will be able to determine if they need this post process.